### DISCUSSION TITLE
webVitalsAttribution not working

### DISCUSSION THREAD

**Discussion by kirchnerto** [vercel/next.js#65048] [Help]
### Summary

Hey everyone,

I'm trying to get the experimental feature "webVitalsAttribution" to working. I followed the instruction to log the webvitals metrics (https://nextjs.org/docs/app/api-reference/functions/use-report-web-vitals) and added the experimental option inside the config like here https://nextjs.org/docs/app/api-reference/next-config-js/webVitalsAttribution.

I managed to get the web vitals logged to the console but still without the attribution add-on.
I tried the same on a boilerplate template here: https://nextjs-demo-five-nu.vercel.app/ (no attribution)

Has anyone got the experimental webVitalsAttribution up and running for debugging lately?

Best wishes,
Tobias

**Comment by danyaridiger** [comment]
What I managed to find out: attribution appears only in `Cumulative Layout Shift` entries regardless of whether the option `webVitalsAttribution` is specified.
<br />
<img width="501" alt="65678934645" src="https://github.com/user-attachments/assets/aab0c6ff-fe01-4829-ac93-7df9de0f5167" />
